src/pages/index.html

```diff
@@ -1,6 +1,197 @@
+            {% endfor %}
+
+        </div>
+    </div>
+</section>
+
+<section id="sec3">
+    <div class="container">
+        <div class="row justify-content-center   ">
+            <div class="col-12 text-center">
+                <h4>130+ Handcoded Blocks</h3>
+                    <h2>Mixland helps you
+                        build beautiful website</h2>
+            </div>
+        </div>
+
+        <div class="row sec3gap justify-content-center  justify-content-lg-between">
+            <div class="col-10  col-lg-4 order-4 order-lg-0">
+                <ul id="seccard" class="cardList d-flex flex-column  ">
+                    {% for item in dataEn.sec3 %}
+                    {{card2.card(item.svgPath,item.title,item.des,item.name,item.state)}}
+                    {% endfor %}
+                </ul>
+
+
+
+            </div>
+            <div class="col-11 col-md-12 col-lg-7 order-0 order-lg-4">
+                <figure class="">
+                    <img src="../assets/images/sec3/Img.jpg" alt="mixland">
+                </figure>
+            </div>
+        </div>
+
+    </div>
+</section>
+
+<section id="sec4">
+    <div class="container">
+        <div class="row text-center text-lg-start align-items-center">
+            <div class="col-lg-6 ">
+                <img src="../assets/images/sec4/customers.png" alt="customers">
+            </div>
+            <div class="col-lg-6 text-center text-lg-start">
+                <h2>Managing & selling digital products is now easier.</h2>
+                <p>Digital products are where it’s at! There are so many benefits to selling digital products. It’s easy
+                    to get started and they can be extremely profitable with smart entrepreneurs and creatives.</p>
+            </div>
+
+        </div>
+    </div>
+</section>
+
+<section id="sec5">
+    <div class="container">
+        <div class="row sec5gap align-items-center">
+            <div class=" col-lg-6 text-center text-lg-start order-6 order-md-0">
+                <div>
+                    <h2>Effectively track real-time and updates from team management</h2>
+                    <p>Designed with a simple, user-friendly interface, nTask is a 7-in-1 project management tool that
+                        is ideal for both small and large teams. You can create multiple workspaces.</p>
+                    <ul class="d-flex flex-column flex-md-row justify-content-center justify-content-lg-start ">
+                        <li>
+                            <h3>12K+ <span>Project Created</span></h3>
+                        </li>
+                        <li>
+                            <h3>1947 <span>Agencies Joined</span></h3>
+                        </li>
+                    </ul>
+                </div>
+            </div>
+            <div class="col-lg-6 order-0 order-md-6">
+                <figure>
+                    <img src="../assets/images/sec5/members.png" alt="members">
+                </figure>
```

Comment:
space between not like the design
remove padding in responsive

Reply:
done